src/MainWindow/main_window.cpp

```diff
@@ -261,9 +261,9 @@ void MainWindow::openProjectDialog(const QString& dir) {
   if (!fileName.isEmpty()) openProject(fileName, false, false);
 }
 
-void MainWindow::closeProject() {
-  if (m_projectManager && m_projectManager->HasDesign() &&
-      confirmCloseProject()) {
+void MainWindow::closeProject(bool force) {
```

Comment:
Why do we need this flag ? close_design could just have been linked to closeProject.

Reply:
because there is a message box. We don't need it in tcl command